admin/app/controllers/spree/admin/digitals_controller.rb

```diff
@@ -17,16 +17,12 @@ def destroy_turbo_stream_enabled?
         true
       end
 
-      def location_after_save
-        spree.admin_product_digitals_url(@product)
-      end
-
       def permitted_resource_params
         params.require(:digital).permit(permitted_digital_attributes)
       end
 
       def permitted_digital_attributes
-        %i[variant_id attachment attachment_file_name attachment_content_type]
+        %i[variant_id attachment]
```

Comment:
@Vegann you don't need to declare this - it's already present in the `PermittedAttributes` module - https://github.com/spree/spree/blob/main/core/lib/spree/permitted_attributes.rb#L68